client-v2/src/util/form.ts

```diff
@@ -101,14 +105,30 @@ export const getInitialValuesForArticleFragmentForm = (
           src: article.imageCutoutSrc,
           width: strToInt(article.imageCutoutSrcWidth),
           height: strToInt(article.imageCutoutSrcHeight),
-          origin: article.imageCutoutSrcOrigin
+          origin: article.imageCutoutSrcOrigin,
+          thumb: article.imageCutoutSrc
         },
         slideshow: slideshow.concat(slideshowBackfill)
       }
     : undefined;
 };
 
+const formToMetaFieldMap: { [fieldName: string]: string } = {
```

Comment:
This is nice and quick to map, but there's some duplication. We could model it the other way round `{[field: string]: Array<string>} `if anyone feels that the grossness factor is too high!